src/Templates/src/templates/maui-blank/.template.config/ide.host.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "icon": "ide/icon.ico",
+    "unsupportedHosts": [
+        {
+            "id": "vs",
+            "version": "(,17.1)"
+        }
+    ]
```

Comment:
```suggestion
    ],
    "requiredComponents": [ 
        { 
              "id": "Microsoft.VisualStudio.ComponentGroup.Maui.All", 
              "hostId": "vs", 
              "componentType": "SetupComponent" 
        }
   ]
```

Reply:
This makes sure the .NET MAUI workload is actually installed for this to show up